provisioning/miniprov/hare_mp/main.py

```diff
@@ -1279,12 +1291,13 @@ def create_parser() -> argparse.ArgumentParser:
                        help_str='Performs the Hare rpm post-upgrade tasks',
                        handler_fn=noop))
 
-    add_service_argument(
```

Comment:
Is this no more required now?

Reply:
It is there, we are just adding one more layer on top of it to support changeset argument.